src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/AssemblyContigAlignmentSignatureClassifier.java

```diff
@@ -97,7 +97,7 @@ private static void decisionTreeForContigsWithTwoAlignments(final JavaRDD<Aligne
         final Tuple2<JavaRDD<AlignedContig>, JavaRDD<AlignedContig>> tandemDupBkptOrSimpleInsDel =
                 RDDUtils.split(strandSwitchOrNot._2, AssemblyContigAlignmentSignatureClassifier::indicatesIntraChrTandemDupBkpts, false);
         contigsByRawTypes.put(RawTypes.InsDel, tandemDupBkptOrSimpleInsDel._2);
-        contigsByRawTypes.put(RawTypes.TandemDupOrMEIBkpt, tandemDupBkptOrSimpleInsDel._1);
+        contigsByRawTypes.replace(RawTypes.TandemDupOrMEIBkpt, contigsByRawTypes.get(RawTypes.TandemDupOrMEIBkpt).union(tandemDupBkptOrSimpleInsDel._1));
```

Comment:
Oh, I see, you fixed this later. I think it might be clearer to keep the first set of alignments in a local variable, and then then only add it to the map after joining the two groups together.

Reply:
done as suggested